### ISSUE TITLE
Bug: Language change not fully working for LA button in ClimateHubs

### ISSUE THREAD

**Issue by tobiasrehm** [climateconnect/climateconnect#1184]
**Describe the bug**
When changing the language in a ClimateHub the language is not fully channging for the contact local ambassador button. Only after refreshing the page it's fully translated.

**To Reproduce**
1. Go to https://climateconnect.earth/de/hubs/erlangen
2. Change language

**Screenshots**
![image](https://user-images.githubusercontent.com/64414429/212885953-4b355a07-415e-4443-b208-3ce96aa9f600.png)

**Comment by positiveimpact** [comment]
The language needs to be as woeking as possible :laughing:

**Comment by positiveimpact** [comment]
Closing in favor of #1492
